### PR TITLE
Dsukha/static site glowup/formatting and grammar

### PR DESCRIPTION
**Pull request by Dave-Sukharan**
There were a few formatting errors that needed to be fixed. Most notably:

1. The sidebar on the Guide page did not reflect the sections
2. The sections on the Guide page were not in the right order
3. Text tagged with the `<code>` tag were formatted strangely
4. Various small grammar/wording fixes on the sub-pages
5. Various grammar/wording fixes on the home page

### Proposed Changes

Items 1, 2, and 4 were largely addressed. Items 3 and 5 are still pending. The text tagged with `<code>` is still not appearing correctly on the Guide page, even after adding the `<pre>` tag.

### Security Implications

- [ ] new software dependencies
- [ ] security controls or supporting software altered
- [ ] new data stored or transmitted
- [ ] security checklist is completed for this change
- [ ] requires more information or team discussion to evaluate security implications
- [x] no PHI/PII is affected by this change

### CODE REVIEW

File: index.html
Comment (epfrankle) [inline index.html:191]: @mtrang1263 - here, and elsewhere, it looks like we've created buttons by styling links as buttons. If so, this is a big miss from an accessibility standpoint. Can we discuss?

File: _includes/data/data_dictionary.html
Comment (epfrankle) [inline _includes/data/data_dictionary.html]: Edited to "in December 2020" in separate commit.

File: _includes/guide/credentials.html
Comment (epfrankle) [inline _includes/guide/credentials.html:8]: @mtrang1263 This is the other accessibility issue I called out in Slack - we need appropriate alt text for the copy to clipboard buttons.

File: terms.html
Comment (epfrankle) [inline terms.html:9]: @mtrang1263 - Is there any way we can comment this page out (or something similar) for launch? Nicole has said that the terms of service not being approved is no longer a blocker, but I'd prefer not to launch with a blank page.

File: _includes/guide/try_the_api.html
Comment (epfrankle) [inline _includes/guide/try_the_api.html]: Would love for this to link to the prod guide page.

**Review by epfrankle — CHANGES_REQUESTED** [review]
Language updates look good. I want to clarify the accessibility questions around button coding and alt-text before approving for merge.

File: _includes/guide/credentials.html
Comment (mtrang1263) [reply _includes/guide/credentials.html:8]: Leveraging `aria-label` to leverage more rich context in the Copy to Clipboard button

File: index.html
Comment (mtrang1263) [reply index.html:191]: Spoke to Elissa offline about this, since we're linking out to another page, ended up using the `<a>` tag and kept the button styling.

File: terms.html
Comment (mtrang1263) [reply terms.html:9]: This was achieved by adding the terms page to our exclusion section. Created this ticket to re-enable when ready: https://jira.cms.gov/browse/BCDA-4024

**Review by epfrankle — APPROVED** [review]
@mtrang1263 @Dave-Sukharan Thank you so much for your incredible work over the past two days to get this ready, and for making the site more accessible to our screen reader users. After we merge this PR, I want to do one final review of the language, and then aside from the Tealium integration we'll be ready to go live.

**Pull request merged**
